bitnami/redis/values.yaml

```
@@ -175,6 +175,76 @@ sentinel:
   ##
   extraEnvVarsSecret: []
 
+  ## Prometheus Exporter / Metrics for Redis Sentinel Exporter
+  ##
+  metrics:
+    enabled: false
+
+    ## Bitnami Redis Sentinel Exporter image
+    ## ref: https://hub.docker.com/r/bitnami/redis-sentinel-exporter/tags/
+    ##
+    image:
+      registry: docker.io
+      repository: bitnami/redis-sentinel-exporter
+      tag: 1.7.1-debian-10-r94
+      pullPolicy: IfNotPresent
+      ## Optionally specify an array of imagePullSecrets.
+      ## Secrets must be manually created in the namespace.
+      ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+      ##
+      # pullSecrets:
+      #   - myRegistryKeySecretName
+
+    ## Sentinel metrics exporter resource requests and limits
+    ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+    ##
+    resources:
+      ## We usually recommend not to specify default resources and to leave this as a conscious
+      ## choice for the user. This also increases chances charts run on environments with little
+      ## resources, such as Minikube. If you do want to specify resources, uncomment the following
+      ## lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+      ## e.g:
+      ## limits:
+      ##   cpu: 500m
+      ##   memory: 1Gi
+      ##
+      limits: {}
+      requests: {}
+
+    ## Extra arguments for Sentinel metrics exporter, for example:
+    ## extraArgs:
+    ##   check-keys: myKey,myOtherKey
+    # extraArgs: {}
+
+    # Enable this if you're using https://github.com/coreos/prometheus-operator
```

```suggestion
    ## Enable this if you're using https://github.com/coreos/prometheus-operator
    ##
```
Try to follow the standard format for comments